### PR TITLE
feat - LibraryPass nft

### PR DESCRIPTION
**Pull request by Olive105**
I added LibraryPass nft. Can you please review and merge? Thanks.

### CODE REVIEW

**Comment by niveda-krish** [comment]
Thanks for submitting this PR - do we plan to support this on testnet? Could you resolve merge conflicts?

File: src/cadence/mainnet/getNFTIDs.cdc [hunk 1]
```diff
@@ -70,9 +70,13 @@ import GoatedGoats from 0x2068315349bdfce5
 import GoatedGoatsTrait from 0x2068315349bdfce5
 import DropzToken from 0x2ba17360b76f0143
 import Necryptolis from 0x718efe5e88fe48ea
+<<<<<<< main
```
Comment (niveda-krish) [inline src/cadence/mainnet/getNFTIDs.cdc:73]: Hi, could you resolve these conflicts?

**Comment by Olive105** [comment]
I resolved

**Comment by Olive105** [comment]
I resolved

**Comment by niveda-krish** [comment]
Hey @Olive105 still seeing a bunch of merge conflicts <<< main

**Comment by niveda-krish** [comment]
Hey @Olive105 reminder to resolve merge conflicts - our biweely release is this Thursday and code will be locked tomorrow

**Comment by niveda-krish** [comment]
Hey @Olive105 reminder to resolve merge conflicts - our biweely release is this Thursday and code will be locked tomorrow

**Review by prpatel05 — CHANGES_REQUESTED** [review]
resolve merge conflicts

**Pull request closed without merging**